code/drasil-example/Drasil/GlassBR/TMods.hs

```diff
@@ -25,16 +25,18 @@ tMods = [pbIsSafe, lrIsSafe]
 
 
 lrIsSafe :: TheoryModel
-lrIsSafe = tm (cw lrIsSafeRC)
+lrIsSafe = tm (cw lrIsSafeQD)
    [qw isSafeLoad, qw tmLRe, qw tmDemand] ([] :: [ConceptChunk])
-   [relToQD locSymbMap lrIsSafeRC] [sy isSafeLoad $= sy tmLRe $> sy tmDemand] [] [makeCite astm2009] 
+   [relToQD locSymbMap lrIsSafeQD] [sy isSafeLoad $= lrIsSafeExpr] [] [makeCite astm2009] 
```

Comment:
So it should be possible to not use `relToQD` here. There was a different combinator in that old branch that was more sensible for use here.  Through this pass, we need to reduce the number of calls to `relToQD` to as close to 0 as possible.

Reply:
My apologies, I overlooked that bit. I just fixed it. The related area in IMods will be using the getEqMods function to grab the QDefinitions there later. I'll do that one in a separate PR (if I can get the logs to not break -- I might need to submit that as an issue instead).